api/server/middleware/version.go

```diff
@@ -67,8 +67,8 @@ func (e versionUnsupportedError) InvalidParameter() {}
 func (v VersionMiddleware) WrapHandler(handler func(ctx context.Context, w http.ResponseWriter, r *http.Request, vars map[string]string) error) func(ctx context.Context, w http.ResponseWriter, r *http.Request, vars map[string]string) error {
 	return func(ctx context.Context, w http.ResponseWriter, r *http.Request, vars map[string]string) error {
 		w.Header().Set("Server", fmt.Sprintf("Docker/%s (%s)", v.serverVersion, runtime.GOOS))
-		w.Header().Set("API-Version", v.defaultAPIVersion)
-		w.Header().Set("OSType", runtime.GOOS)
+		w.Header().Set("Api-Version", v.defaultAPIVersion)
+		w.Header().Set("Ostype", runtime.GOOS)
```

Comment:
For others  reviewing; I suggested changing these to match the normalised  version.  Effectively these  parts of the change are a "no-op", but explicitly setting the headers in  the normalised form (instead of them implicitly being normalised by Go stdlib) reduces confusion.

As a follow-up to this PR I want to have  a  peek if we could define consts for this  (on which we could document why they're capitalised as they are), but I have not yet looked at the impact of that (i.e.; where to best put the const without that resulting in cumbersome imports / inter-package dependencies).